dartdoc.iml

```diff
@@ -33,7 +33,7 @@
       <excludeFolder url="file://$MODULE_DIR$/tool/packages" />
     </content>
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="Dart Packages" level="project" />
     <orderEntry type="library" name="Dart SDK" level="project" />
+    <orderEntry type="library" name="Dart SDK" level="application" />
```

Comment:
Currently the goobuntu version is 2016.3, tried rerunning install script but that changed nothing AFAICT.